app/event-creator.tsx

```diff
@@ -1,14 +1,63 @@
 'use client'
 
-import React from "react";
+import React, { useState } from "react";
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
 
 export default function EventCreator() {
 
+   
+
     const [name, setName] = React.useState("");
     const [date, setDate] = React.useState("");
     const [calendarId, setCalendarId] = React.useState("");
 
+
+    function validateInput() {
+
+        var valid = true;
+
+        if (name.length == 0) {
+            const el = document.getElementById("name");
+            el.style.border = "2px solid red";
```

Comment:
this string repeats 3 times in this function. Couldn't we extract it in a variable?